.gitignore

```diff
@@ -1,7 +1,9 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+
 # Jetbrains Products
 .idea
+.rollup.cache
```

Comment:
plz remove this line because it is not part of these changes. This belongs to the branch `feat/roll-up`

Reply:
Done!